WebContent/AdminLogin.html

```diff
@@ -120,6 +166,13 @@ <h1>Welcome to Online Book Store</h1>
 				<form action="adminlog" method="post">

```

Comment:
a) Add an id for this form, theform and also add the onsubmit property
b) add <br> at top and bottom of the captcha line
c) add the placeholder in captcha

check this commit changes: https://github.com/shashirajraja/onlinebookstore/pull/58/commits/65cabe5524ecc79cff553966fbdbffff22f7e4dc